lib/ansible/executor/task_executor.py

```diff
@@ -259,7 +259,13 @@ def _run_loop(self, items):
             # the value may be 'None', so we still need to default it back to 'item'
             loop_var = self._task.loop_control.loop_var or 'item'
             label = self._task.loop_control.label or ('{{' + loop_var + '}}')
-            loop_pause = self._task.loop_control.pause or 0
+            # Check if a variable has been passed as the 'pause' var
+            if self._task.loop_control.pause is not None and not isinstance(self._task.loop_control.pause,float):
+                templar = Templar(loader=self._loader, shared_loader_obj=self._shared_loader_obj, variables=self._job_vars)
+                loop_pause = float(templar.template(self._task.loop_control.pause))
+            else:
+                loop_pause = self._task.loop_control.pause or 0
```

Comment:
Ah ok, the label templating was working, see here https://github.com/PhilipGough/ansible/blob/18a9b35070852b95f6472f0bce08a3a1ea315e07/lib/ansible/executor/task_executor.py#L308